Default to small icons and Sans font.
Lubomir Rintel <lkundrak@v3.sk>

--- inkscape-0.46+devel/src/preferences-skeleton.h.icons	2008-06-24 13:38:52.000000000 +0200
+++ inkscape-0.46+devel/src/preferences-skeleton.h	2008-07-05 04:50:19.000000000 +0200
@@ -86,7 +86,7 @@
 "    </eventcontext>\n"
 "    <eventcontext id=\"text\"  usecurrent=\"0\" gradientdrag=\"1\"\n"
 "                       font_sample=\"AaBbCcIiPpQq12369$\342\202\254\302\242?.;/()\"\n"
-"                  style=\"fill:black;fill-opacity:1;stroke:none;font-family:Bitstream Vera Sans;font-style:normal;font-weight:normal;font-size:40px;\" selcue=\"1\"/>\n"
+"                  style=\"fill:black;fill-opacity:1;stroke:none;font-family:Sans;font-style:normal;font-weight:normal;font-size:40px;\" selcue=\"1\"/>\n"
 "    <eventcontext id=\"nodes\" selcue=\"1\" gradientdrag=\"1\" highlight_color=\"4278190335\" pathflash_enabled=\"1\" pathflash_timeout=\"500\" show_handles=\"1\" show_helperpath=\"0\" sculpting_profile=\"1\" />\n"
 "    <eventcontext id=\"tweak\" selcue=\"0\" gradientdrag=\"0\" show_handles=\"0\" width=\"0.2\" force=\"0.2\" fidelity=\"0.5\" usepressure=\"1\" style=\"fill:red;stroke:none;\" usecurrent=\"0\"/>\n"
 "    <eventcontext id=\"gradient\" selcue=\"1\"/>\n"
@@ -327,7 +327,7 @@
 "    <group\n"
 "       id=\"tools\"\n"
 "       icononly=\"1\"\n"
-"       small=\"0\" />\n"
+"       small=\"1\" />\n"
 "  </group>\n"
 "\n"
 "  <group\n"
